lib/radiator_web/controllers/admin/usersettings_controller.ex

```diff
@@ -0,0 +1,45 @@
+defmodule RadiatorWeb.Admin.UserSettingsController do
+  use RadiatorWeb, :controller
+
+  alias Radiator.Auth
+
+  action_fallback RadiatorWeb.Api.FallbackController
+
+  def index(conn, _params) do
+    render(conn, "usersettings.html", changeset: Auth.Register.change_user(%Auth.User{}))
+  end
+
+  def create(conn, params) do
+    user_map = params["user"]
```

Comment:
`user_params` is the more common variable name choice here. Variable type is not usually included in the name.